compiler/src/iree/compiler/InputConversion/StableHLO/Preprocessing/StableHLOToStableHLO.cpp

```diff
@@ -1275,6 +1276,9 @@ struct StableHLOToStableHLOPreprocessing final
     }
 
     RewritePatternSet patterns(context);
+    // General canonicalization patterns.
+    populateCanonicalizationPatterns(context, &patterns, PatternBenefit{1024});
```

Comment:
Why is benefit needed?

Reply:
These canonicalizations are always more profitable than what the other patterns in this pass produce. It will be more difficult to undo expansions into other ops and eliminate those later.

I'll add a comment explaining this.